coursedata/adventures/de.yaml

```diff
@@ -460,119 +485,142 @@ adventures:
 
-                    After Baby Shark you can of course also program other songs. There are many songs with repetition.
+                    Du kannst natürlich auch andere Lieder programmieren.
+                    In vielen Liedern wiederholt sich etwas.
+                    Fällt Dir eins ein?
+
                 start_code: "print 'Baby Shark'"
             6:
                 story_text: |
-                    ## Songs
-                    Songs often contain a lot of repetition. Sometimes the repetition is also based on counting.
-                    For example, in the well-known song 'Bottles of beer'. You can program that song with a little math.
+                    ## Lieder
+                    In Liedern wiederholt sich oft einiges.
+                    Manchmal beinhaltet die Wiederholung auch, dass etwas gezählt wird.
+                    So zum Beispiel beim Lied über die Bierflaschen.
+                    Dieses Lied kannst Du mit einem kleinen bisschen Mathematik einfach programmieren.
 
-                    ## Example Hedy code
+                example_code : |
+                    ## Hedy Beispiel-Code
                     ```
-                    verse = 99
-                    print verse ' bottles of beer on the wall'
-                    print verse ' bottles of beer'
-                    print 'Take one down, pass it around'
-                    verse = verse - 1
-                    print verse ' bottles of beer on the wall'
+                    Strophe = 99
+                    print Strophe ' Flaschen Bier an der Wand,'
+                    print Strophe ' Flaschen Bier.'
+                    print 'Nimm eine runter, reich sie herum,'
+                    Strophe = Strophe - 1
+                    print Strophe ' Flaschen Bier an der Wand.'
                     ```
 
-                    You can now repeat lines 2 to 9 as many times as you want by copying the lines.
+                    Du kannst jetzt die Zeilen 2 bis 6 so oft wiederholen, wie Du möchtest.
+                    Dazu kopierst Du sie und fügst sie am Ende wieder ein.
 
-                start_code: "print 'Baby shark'"
+                start_code: "print 'Baby Shark'"
             16:
                 story_text: |
-                    ## Sing a song!
-                    In this level, you can program a song like OldMacDonald even more quickly. You can connect the right animal to the right sound by simply putting them in the same place in the list.
-                    The Drunken Sailor is also quickly made in this level. You only need 8 lines for the entire song, check it out!
+                    ## Lieder
+                    In diesem Level kannst Du 'Old MacDonald hat ne Farm' sogar noch schneller programmieren.
+                    Du kannst jedem Tier das richtige Geräusch zuordnen, indem Du sie an die selbe Stelle einer Liste speicherst.
+                    Wenn Du schon Englisch sprichst, kennst Du vielleicht das Lied 'The Drunken Sailor'.
```

Comment:
I struggled with this one as well, because I pretty much agree with you. I think, I learned it when I was somewhere around 7th grade. I just asked my daughter who is in 5th. She knows it - because I keep singing it :)
Anway, I like your suggestion. I'll take it.